README

```diff
@@ -0,0 +1,16 @@
+Pull various Juju KPI data in and report on it
+=====
+
+Long running instance data
+------
+
+You need to get the log files onto the system. For this you need credentials
+to the charmstore logs via swift.
+
+  make sysdeps
+  make
+  source ~/path/to/novarc/for/swift
+  make get-logs
+  make _initdb
+  make longrunning
```

Comment:
OK, I got this far. Everything worked.

BUT...

what the hell was the point? What am I supposed to see? Can you beef up the README to describe in better detail what the project is for, how to see the results, etc?